src/main/kotlin/no/nav/familie/ef/sak/behandling/revurdering/AutomatiskRevurderingController.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ef.sak.behandling.revurdering
+
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.slf4j.LoggerFactory
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/api/revurdering"])
```

Comment:
Når er denne lik som for `RevurderingController` kunne man endret til `automatisk-revurdering` for eksempel?